pkgs/dartpad_ui/README.md

```diff
@@ -37,3 +37,40 @@ There are options to run UI locally:
     flutter run -d chrome --web-port 8888 --web-browser-flag "--disable-web-security" \
       --web-launch-url=http://localhost:8888/?channel=localhost
     ```
+
+## How to publish
+
+If you want to collaborate on intermediate version, you can publish it to your own Firebase project:
```

Comment:
```suggestion
If you want to collaborate on an intermediate version, you can publish it to your own Firebase project:
```

Reply:
Oh, I thought disabled button means it is applied.

https://github.com/dart-lang/dart-pad/pull/3234